cdci_data_analysis/analysis/parameters.py

```diff
@@ -473,9 +480,12 @@ def __init__(self, value=None, delta_T_format='sec', name=None, delta_T_format_n
                          default_units='sec',
                          name=name)
 
-    def get_value_in_default_format(self) -> Union[str, float, None]:
+    def get_value_in_default_units(self) -> Union[str, float, None]:
         return getattr(self._astropy_time_delta, self.default_units)
 
```

Comment:
```suggestion

    def get_value_in_default_format(self):
        return self.get_value_in_default_units()

```
To ensure backward compatibility. Also, _format_ for time sounds more natural than _units_